plugins/gatsby-plugin-security-bulletins-rss/gatsby-node.js

```diff
@@ -0,0 +1,130 @@
+const fs = require('fs');
+const path = require('path');
+const RSS = require('rss');
+const format = require('date-fns/format');
+const html = require('rehype-stringify');
+const unified = require('unified');
+const toHast = require('mdast-util-to-hast');
+const removeImports = require('remark-mdx-remove-imports');
+const removeExports = require('remark-mdx-remove-exports');
+const addAbsoluteImagePath = require('../../rehype-plugins/utils/addAbsoluteImagePath');
+
+const htmlGenerator = unified()
+  .use(removeImports)
+  .use(removeExports)
+  .use(addAbsoluteImagePath)
+  .use(html);
+
+const securityBulletinsQuery = async (graphql) => {
+  const query = `
+    {
+      site {
+        siteMetadata {
+          title
+          siteUrl
+        }
+      }
+      securityBulletinFileNodes: allMdx(
+        filter: {fileAbsolutePath: {regex: "/src/content/docs/security/.*/security-bulletins/.*(?<!index).mdx/"}}
+      ) {
+        nodes {
+          frontmatter {
+            title
+            metaDescription
+          }
+          slug
+          mdxAST
+        }
+      }
+    }
+    `;
+
+  const { data } = await graphql(query);
+
+  return data;
+};
+
+const getFeedItem = (node, siteMetadata) => {
+  const { frontmatter, slug, mdxAST } = node;
+  const { title, metaDescription } = frontmatter;
+
+  const transformedAST = htmlGenerator.runSync(mdxAST);
+  const html = htmlGenerator.stringify(toHast(transformedAST));
+
+  // time is necessary for RSS validity
+  const date = new Date();
+  const pubDate = `${format(date, 'EE, dd LLL yyyy')} 00:00:00 +0000`;
+  const link = new URL(slug, siteMetadata.siteUrl).href;
+  const id = Buffer.from(`${title}`).toString('base64');
```

Comment:
We might want to add the date to help insure uniqueness:
```suggestion
  const id = Buffer.from(`${date}-${title}`).toString('base64');
```

Reply:
unfortunately this date is going to reflect the current date/time when it builds, so it'll change each time which seemed bad. additionally, the `title` for most bulletins contains a unique ID for a finding. for example: `Security Bulletin NR20-01` which is two digit year and the number of the finding for that year.

Reply:
i originally had it using the file `birthTime`, but since the builds copy files over those dates will also always reflect the date/time of the build. i couldn't find any other way to reference the date/time the file was created, short of adding it for frontmatter.